### DISCUSSION TITLE
SQLAlchemy 2.0??

### DISCUSSION THREAD

**Discussion by phyyou** [apache/airflow#27682] [Q&A] [answered]
Hi!
I have a just simple question.

Did you have a plan for upgrade sqlalchemy version 1.4 to 2.0 in airflow metadata database?

**Comment by potiuk** [comment]
If someone does it - why not.

**Comment by potiuk** [comment, marked as answer]
More info: For now 2.0 is beta, and likely breaking a lot of things - so this is likely quite an effort. But - as usual in the OSS - if someone takes the task on their shoulder and leads it to completion, then it will be done. It can be anyone - including yourself @phyyou  - so if you are up to the task - you can even start testing and running a PR that will succeed once all tests pass and we can consider merging it.

**Reply by phyyou** [reply]
Thanks 👍
